sdk/dnsresolver/arm-dnsresolver/CHANGELOG.md

```diff
@@ -1,15 +1,119 @@
+  - Added Interface DnsSecurityRulesListOptionalParams
+  - Added Interface DnsSecurityRulesUpdateHeaders
+  - Added Interface DnsSecurityRulesUpdateOptionalParams
+  - Added Interface ErrorAdditionalInfo
+  - Added Interface ErrorDetail
+  - Added Interface ErrorResponse
+  - Added Type Alias Action
+  - Added Type Alias ActionType
+  - Added Type Alias DnsResolverDomainListsBulkResponse
+  - Added Type Alias DnsResolverDomainListsCreateOrUpdateResponse
+  - Added Type Alias DnsResolverDomainListsDeleteResponse
+  - Added Type Alias DnsResolverDomainListsGetResponse
+  - Added Type Alias DnsResolverDomainListsListByResourceGroupNextResponse
+  - Added Type Alias DnsResolverDomainListsListByResourceGroupResponse
+  - Added Type Alias DnsResolverDomainListsListNextResponse
+  - Added Type Alias DnsResolverDomainListsListResponse
+  - Added Type Alias DnsResolverDomainListsUpdateResponse
+  - Added Type Alias DnsResolverPoliciesCreateOrUpdateResponse
+  - Added Type Alias DnsResolverPoliciesDeleteResponse
+  - Added Type Alias DnsResolverPoliciesGetResponse
+  - Added Type Alias DnsResolverPoliciesListByResourceGroupNextResponse
+  - Added Type Alias DnsResolverPoliciesListByResourceGroupResponse
+  - Added Type Alias DnsResolverPoliciesListByVirtualNetworkNextResponse
+  - Added Type Alias DnsResolverPoliciesListByVirtualNetworkResponse
+  - Added Type Alias DnsResolverPoliciesListNextResponse
+  - Added Type Alias DnsResolverPoliciesListResponse
+  - Added Type Alias DnsResolverPoliciesUpdateResponse
+  - Added Type Alias DnsResolverPolicyVirtualNetworkLinksCreateOrUpdateResponse
+  - Added Type Alias DnsResolverPolicyVirtualNetworkLinksDeleteResponse
+  - Added Type Alias DnsResolverPolicyVirtualNetworkLinksGetResponse
+  - Added Type Alias DnsResolverPolicyVirtualNetworkLinksListNextResponse
+  - Added Type Alias DnsResolverPolicyVirtualNetworkLinksListResponse
+  - Added Type Alias DnsResolverPolicyVirtualNetworkLinksUpdateResponse
+  - Added Type Alias DnsSecurityRulesCreateOrUpdateResponse
+  - Added Type Alias DnsSecurityRulesDeleteResponse
+  - Added Type Alias DnsSecurityRulesGetResponse
+  - Added Type Alias DnsSecurityRulesListNextResponse
+  - Added Type Alias DnsSecurityRulesListResponse
+  - Added Type Alias DnsSecurityRuleState
+  - Added Type Alias DnsSecurityRulesUpdateResponse
+  - Interface Resource has a new optional parameter systemData
+  - Added Enum KnownAction
+  - Added Enum KnownActionType
+  - Added Enum KnownDnsSecurityRuleState
 
-### Other Changes
+### Breaking Changes
 
+  - Interface DnsForwardingRuleset no longer has parameter systemData
+  - Interface DnsResolver no longer has parameter systemData
+  - Interface ForwardingRule no longer has parameter systemData
+  - Interface InboundEndpoint no longer has parameter systemData
+  - Interface OutboundEndpoint no longer has parameter systemData
+  - Interface VirtualNetworkLink no longer has parameter systemData
```

Comment:
Could you pin swagger link to me?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/34869#issuecomment-2901889473